src/routes/transactions/entities/transfer-transaction-info.entity.ts

```diff
@@ -35,3 +35,9 @@ export class TransferTransactionInfo extends TransactionInfo {
     this.transferInfo = transferInfo;
   }
 }
+
+export function isTransferTransactionInfo(
+  txInfo: TransactionInfo,
+): txInfo is TransferTransactionInfo {
+  return txInfo.type === 'Transfer';
```

Comment:
Out of scope here, but maybe we should consider creating an enum for the `type` of `TransactionInfo`, should there be any changes to the `type` value. What do you think?